packages/components/components/index.ts

```diff
@@ -18,14 +25,20 @@ export {
 
   Text,
 
+  Blurb,
   Box,
   Button,
+  Checkbox,
   ConvertKit,
+  FeatureHighlight,
+  Footer,
+  HandleTag,
   Hero,
   Input,
   Label,
 
   Modal,
+  Navigation,
   Textarea,
   Wrapper,
 };
```

Comment:
I think there's a way to auto generate this file. Food for thought in the future